Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/.template.config/template.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Wilderness Labs",
+  "classifications": [ "Meadow", "Library" ],
+  "name": "Meadow StartKit (Core)",
+  "identity": "WildernessLabs.Meadow.StartKit.Core",
+  "groupIdentity": "WildernessLabs.Meadow.SKC",
+  "shortName": "MeadowSKC",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "sourceName": "$safeprojectname$",
```

Comment:
@jorgedevs 1. if we want our project templates to work with the --name parameter, then we need to add this as "sourcename", then rename our csproj files to 2. and add $safeprojectname$ to any .cs files (3.) where we want that replaced.